tests/test_schema31.py

```diff
@@ -37,9 +37,7 @@ def test_example_json_validates(example_json):
 def test_json_to_xml(example_xml_file, example_json, xsd31):
     """Test that example XML converts to example JSON."""
     xsd31.assertValid(etree.XML(example_xml_file.encode('utf8')))
-    assert ET.dump(ET.XML(example_xml_file)) == \
-        ET.dump(ET.XML(tostring(example_json)))
```

Comment:
this does nothing as the `ET.dump` function returns nothing. see https://docs.python.org/2/library/xml.etree.elementtree.html#xml.etree.ElementTree.dump
it is like testing `None == None`